plugins/gfm/build.gradle.kts

```diff
@@ -12,7 +12,7 @@ publishing {
 configureBintrayPublication("gfmPlugin")
 
 dependencies {
-    compileOnly(project(":plugins:base"))
+    implementation(project(":plugins:base"))
```

Comment:
I'm not sure why this change is needed, could you please explain?

Reply:
Because it needs the base plugin at runtime and compile time. I am not sure why it was different before!

Reply:
Not really, dokka gradle task adds dokka base in the runtime, so this way we end having the same dependency (dokka base) twice

Reply:
No, the dokka task just adds a "default" dependency (which is necessary, because otherwise you could not have a dokka task **without** base). Since this tasks depends on base, it should declare its dependency on base!

Reply:
This way you won't be able to have a version difference between the plugin and the base, I'm now wondering if that's an issue though. 
You may want to have one version different than the other and while it should be possible to do with Gradle configuration I'm wondering if we should make it easier or not  🤔